x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/inference/ltr/LearningToRankRescorerBuilderRewriteTests.java

```diff
@@ -218,6 +218,24 @@ public void testBuildContext() throws Exception {
         );
     }
 
+    public void testBuildContextWindowSize() throws IOException {
+        int windowSize = randomInt();
```

Comment:
We may want to make this more specific, such as something like `randomNonNegativeInt()` or `randomIntBetween(1, 10000)`

Reply:
This tests have been reverted because we changed our approach and are now checking for the window size during the parsing instead of doing it during context building.